akka-actor/src/main/scala/akka/pattern/Patterns.scala

```diff
@@ -48,6 +48,37 @@ object Patterns {
   def ask(actor: ActorRef, message: Any, timeout: Timeout): Future[AnyRef] = scalaAsk(actor, message)(timeout).asInstanceOf[Future[AnyRef]]
 
   /**
+   * A variation of ask which allows to implement "replyTo" pattern by including
+   * sender reference in message.
+   *
+   * For use with Akka compiled with Scala 2.12.
```

Comment:
Hmm, it would be nice to just do “the right thing” for each release, but that might be impossible. Given that `japi.Function` works fine independent of Scala version I’d remove this variant without replacement—in a different universe this would be the only one we need …